ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -304,7 +304,7 @@ tests:
     test:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi
-- as: aws-ipi-f9-longduration-mco-all
+- as: aws-ipi-longduration-mco-g1-f9
```

Comment:
can we have job name here  mentioned `high` ? As here we are just looking for high test-case it could make easy to identify too, what do you think about this?

Reply:
okay, got it ! Thankyou
everything looks good to me. Adding lgtm label